logic/nodes.go

```diff
@@ -256,6 +256,7 @@ func SetNodeDefaults(node *models.Node) {
 		postdown := parentNetwork.DefaultPostDown
 		node.PostDown = postdown
 	}
+	node.Connected = true
```

Comment:
this is not a good way to handle this also doesn't affect mq, should be discussed and moved to other PR